src/sestest/VerifyCommand.cs

```diff
@@ -38,24 +38,16 @@ public VerifyCommand(ILogger logger) : base(logger)
         /// <returns>Results of execution (0 = success).</returns>
         public async Task<int> Execute(VerifyCommandLine commandLine)
         {
-            Errorable<Uri> server = FindServer(commandLine);
-            Errorable<string> token = FindToken(commandLine);
-            Errorable<string> app = FindApplication(commandLine);
-            Errorable<string> api = FindApiVersion(commandLine);
-
-            Errorable<int> result = await server.With(token)
-                .With(app)
-                .With(api)
-                .MapAsync(SubmitToken)
-                .ConfigureAwait(false);
-
-            return result.Match(
-                exitCode => exitCode,
-                errors =>
-                {
-                    Logger.LogErrors(errors);
-                    return ResultCodes.Failed;
-                });
+            var exitCodeResult = await
+                (
+                from server in FindServer(commandLine)
+                join token in FindToken(commandLine) on true equals true
+                join app in FindApplication(commandLine) on true equals true
+                join api in FindApiVersion(commandLine) on true equals true
+                select SubmitToken(server, token, app, api)
+                ).AsTask().ConfigureAwait(false);
```

Comment:
Would this work as well?

```
from server in FindServer(commandLine)
from token in FindToken(commandLine)
from app in FindApplication(commandLine)
from api in FindApiVersion(commandLine)
select SubmitToken(server, token, app, api)
```

Reply:
It would work, but with different behaviour. Only the errors from `FindServer` would be logged, and none of the other command-line validation would happen.

The point of using `join` is to allow errors to be combined, which is typically what you want in input validation scenarios like this.